### ISSUE TITLE
My audio is having problems when using the OpenAI add-on

### ISSUE THREAD

**Issue by MuhammadGagah** [aaclause/nvda-OpenAI#28]
Hello, I am using OpenAI-23.12.10-72b019c, everything is running smoothly, but the issue I am encountering is that when NVDA restarts, my TeamTalk audio cuts out, it seems like this add-on is refreshing the audio. I am not sure what the core of the problem is, but when I disable this add-on and restart NVDA, there are no audio issues that arise. I also have not conducted audio tests such as in a music player and so on.
Can you fix this?

**Comment by MuhammadGagah** [comment]
No, this happened even when I wasn't using a microphone recording. But it seems that it doesn't happen anymore with the file you provided above, I will investigate further and will close the issue if it's really resolved.

**Comment by MuhammadGagah** [comment]
Oh no, it seems that the problem still occurs with the file you provided above, but there is a change. My Audio TeamTalk still crashes when I restart NVDA with this add-on, but after a while, the audio comes back on. This add-on appears to refresh the audio, I wonder why that is?

**Comment by MuhammadGagah** [comment]
I have already tried, and the problem remains the same. Can you investigate this issue further?

**Comment by MuhammadGagah** [comment]
I haven't tried it on NVDA 2024. This is very strange, the problem occurs only when restarting NVDA, but if I do Reload plugins this audio problem does not occur.

**Comment by blindpk** [comment]
@aaclause I have the exact same problem. When I start NVDA, all audio cuts out for maybe a second (including NVDA's own) and then comes back. I have the addon on two different computers and the problem seems to only occur on one of them, but I will investigate further if I can find out more details. I don't use recording either, but maybe there some kind of initialization going on?
